### ISSUE TITLE
Skip to navigation link does not work on mobile layout

### ISSUE THREAD

**Issue by TrebBrennan** [govau/designsystem#68]
RE: #54 

# What happened
In mobile layout
- tab to 'skip to navigation' and enter
- browser focus jumps down page to below menu toggle (see #67)

# Expected
In mobile layout
- tab to 'skip to navigation' and enter
- browser focus jumps to 'open menu'
